docs/content/mobile/android/cloud-deploy.md

```diff
@@ -7,30 +7,45 @@ Before reading this document, make sure you are familiar with:
 
+## Prepare your project to connect to a cloud deployment {#prepare}
+
+To connect your Android device to a cloud deployment, you need a mobile connector script.
+
+<%(Callout message="If you are using one of our Starter Projects, you can skip the **Create a mobile connector script** section below, as you already have one in your project.")%>
+
+<%(#Expandable title="Create a mobile connector script")%>
+
+If you [added the GDK]({{urlRoot}}/content/set-up-new-project) to an existing Unity project rather than using a Starter Project, then you also need to create and add a MonoBehaviour script to your Android client-worker GameObject. To do this:
+
+1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want. You can base your implementation on [the one](https://github.com/spatialos/gdk-for-unity-fps-starter-project/blob/master/workers/unity/Assets/Fps/Scripts/SetupLogic/AndroidWorkerConnector.cs) in our FPS Starter Project.
+1. In your Unity Editor, add the MonoBehaviour script to your Android client-worker GameObject.
+1. In your Unity Editor, navigate to your Android client-worker GameObject and ensure the `ShouldConnectLocally` checkbox is **not** checked in the script's drop-down window of the Inspector window.
+
+<%(/Expandable)%>
+
+## Android device or Android emulator
+
 To connect your mobile application to a cloud deployment, you need to authenticate against our services.
 This guide describes how to authenticate using the development authentication flow which we provide for the early stages of game development.
 Altenatively, if you want to create your own authentication server, follow [this guide](https://docs.improbable.io/reference/latest/shared/auth/integrate-authentication-platform-sdk).
 
-## Connecting your Android device or emulator to a cloud deployment
-
 1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
 1. [Start your Android Emulator in Android Studio](https://developer.android.com/studio/run/managing-avds) or connect your Android device to your development computer.
 
     > If you start an emulator, ensure you choose the same CPU architecture for your virtual machine as your development computer. Using a different architecture might affect the performance of your emulator.
 
-1. [Build your server-worker types.]({{urlRoot}}/content/build)
-1. Upload your server-worker types. To do this, open a terminal window and from the root directory of your SpatialOS project, enter `spatial cloud upload <assembly name>`.
+1. [Build your server-workers.]({{urlRoot}}/content/build)
```

Comment:
that was a change made deliberatly by @ElleEss .I think they want to use worker instances and worker types everywhere. So double-check with her first before merging it in :)

Reply:
@NotRobSheridan says:

>For this particular line, if we don’t need to distinguish between different worker types, and if they’re not currently running in a deployment, we can stick to `server-workers`.

>we’re saving `server-worker instances` to refer to server workers that are currently running in a deployment.